compiler/passes/scopeResolve.cpp

```diff
@@ -1939,11 +1953,14 @@ static bool lookupThisScopeAndUses(const char*           name,
           // symbols that they define.
           forv_Vec(VisibilityStmt, stmt, *moduleUses) {
             if (stmt != NULL) {
-              if (Symbol* modSym = stmt->checkIfModuleNameMatches(name)) {
-                if (isRepeat(modSym, symbols) == false) {
-                  symbols.push_back(modSym);
-                  if (storeRenames && stmt->isARename()) {
-                    renameLocs[modSym] = &stmt->astloc;
+              if (!isImportStmt(stmt) ||
+                  toImportStmt(stmt)->providesQualifiedAccess()) {
```

Comment:
This seems surprising/wrong to me. If we have `import M.x;`, I am *not* expecting that `M` be available as a symbol. You would have to do `import M;` for that.

Reply:
Oh, wait, I think I was misreading this code. It says providesQualifiedAccess but I think I read the opposite.

Reply:
That's what this case covers.  `providesQualifiedAccess` returns true for `import M;` and false for `import M.x;`

Reply:
Just saw your second comment, sorry.  It's easy to do :)